utils/array-utils.ts

```diff
@@ -0,0 +1,9 @@
+export const getPagesArray = (lastPageNumber: number, length?: number): readonly string[] => {
+  const fullArray = Array.from({ length: lastPageNumber + 1 }, (_val, index) =>
+    index.toString(),
```

Comment:
Mamy stronę numer 0? Nie powinno być przypadkiem 
```suggestion
    index + 1,
```